backstory/src/main/webapp/script.js

```diff
@@ -153,3 +220,16 @@ function updateFileName() {
     }
   }
 }
+
+/**
+ * Enables the submit button if there's a file in
+ * the image-upload field.
+ */
+function enableSubmitButton() {
+  const fileInput = document.getElementById('image-upload');
+  const submitButton = document.getElementById('submit-button');
+
+  if (fileInput.files.length > 0) {
```

Comment:
Could we use validateImageUpload here? That would get us the type checking of validateImageUpload and if the validation logic ever changes in the future it would keep the two pieces of code in sync.

Reply:
I can definitely add this. Thank you for the suggestion!

Reply:
Done